### DISCUSSION TITLE
Overhang threshold Bridge flow

### DISCUSSION THREAD

**Discussion by tome9111991** [supermerill/SuperSlicer#1630] [Q&A] [answered]
hello everyone.
can someone tell me what effect the setting overhangs bridge flow has on the print quality?
![Screenshot 2021-10-10 212124](https://user-images.githubusercontent.com/57866234/136710293-9620c3e5-887f-467c-8ff0-8fa6b94655b5.png)

understand the sense of the setting not really the flow of overhangs is changed.
not sure if this is useful.

**Comment by supermerill** [comment, marked as answer]
Bridge flow is used to output a round extrusion when there is nothing under it to flatten it.
This setting control when to use this behaviour for overhangs.
Before, it was hardcoded to something like 50%, for the fan & bridge (if you activate overhang detection).
Now, you can control it.
